helm/hpcc/templates/localroxie.yaml

```diff
@@ -12,14 +12,16 @@ data:
     roxie:
 {{ toYaml (omit .me "logging") | indent 6 }}
 {{- include "hpcc.generateLoggingConfig" . | indent 6 }}
-{{ include "hpcc.generateVaultConfig" (dict "root" .root "categories" (list "storage" "ecl" "ecl-user" ) ) | indent 6 }}
+{{ include "hpcc.generateVaultConfig" (dict "root" .root "categories" .secretsCategories ) | indent 6 }}
     global:
 {{ include "hpcc.generateGlobalConfigMap" .root | indent 6 }}
 {{- end -}}
 
 {{ range $roxie := $.Values.roxie -}}
 {{- if not $roxie.disabled  -}}
-{{- $commonCtx := dict "root" $ "me" $roxie }}
+{{- $secretsCategoriesUser := list "storage" "ecl" "ecl-user" }}
+{{- $secretsCategoriesSystem := list "system" "ecl" "ecl-user" }}
```

Comment:
I don't think it make sense for these to differ

Reply:
why would secretsCategoriesSystem not have "storage" (for example)?

Reply:
or why do both have "ecl-user" here, but in eclagent only secretsCategoriesUser does?